# This message contains all the distances published by the arduino
float64 Frontleft
float64 Frontright
float64 arm

